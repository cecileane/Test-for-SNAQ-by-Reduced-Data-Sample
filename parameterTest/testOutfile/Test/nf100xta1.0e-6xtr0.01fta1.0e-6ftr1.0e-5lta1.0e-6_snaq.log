optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 100, liktolAbs = 1.0e-6.
rootname for files: nf100xta1.0e-6xtr0.01fta1.0e-6ftr1.0e-5lta1.0e-6_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 19907
seed: 19907 for run 1
Sat Oct  1 17:06:51 2016

 BEGIN SNaQ for run 1, seed 19907 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [0,25,25,4,0,61], Nmov [217,25,25,4,10000,61]
END optTopLevel: found minimizer topology at step 148 (failures: 97) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 34		 34		 2	 |	 1		 1		 --
mvtarget	 29		 29		 1	 |	 1		 1		 --
chdir		 10		 10		 1	 |	 2		 2		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 73		 52		 1	 |	 1		 1		 --
Total		 147		 126		 6	 |	 5		 5		 3
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 1, -loglik of best 155.49625734551452
(3,(6,5):0.335347317849328,((1,(2)#H7:::0.6269753195260324):2.155860132049398,(4,#H7:::0.3730246804739677):2.155860132049398):0.12573310842051721);
---------------------
seed: 51950 for run 2
Sat Oct  1 17:06:59 2016

 BEGIN SNaQ for run 2, seed 51950 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 114 (failures: 100) with -loglik=169.89573 and ht_min=[0.39462,0.2228,0.06849,1.13004,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 26		 26		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 5		 5		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 57		 47		 1	 |	 5		 5		 --
Total		 114		 104		 2	 |	 5		 5		 3
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 2, -loglik of best 169.89572985762143
((3,(1,(2,#H7:::0.39462115396102704):10.0):1.130042730052587):0.22279535041049814,6,(5,(4)#H7:::0.605378846038973):0.06848863698245954);
---------------------

Sat Oct  1 17:07:07 2016
MaxNet is (3,(6,5):0.335347317849328,((1,(2)#H7:::0.6269753195260324):2.155860132049398,(4,#H7:::0.3730246804739677):2.155860132049398):0.12573310842051721); 
with -loglik 155.49625734551452
