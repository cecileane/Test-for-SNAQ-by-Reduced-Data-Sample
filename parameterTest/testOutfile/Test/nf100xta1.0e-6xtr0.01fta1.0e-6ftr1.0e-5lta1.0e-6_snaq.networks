(3,(6,5):0.335347317849328,((1,(2)#H7:::0.6269753195260324):2.155860132049398,(4,#H7:::0.3730246804739677):2.155860132049398):0.12573310842051721);, with -loglik 155.49625734551452 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,(6,5):0.32586713982689336,((1,(#H7:::0.3180905287548815,2):7.818667027426831):1.1159915404848715,(4)#H7:::0.6819094712451186):0.0);, with -loglik 158.62922220111065
(3,(6,5):0.33241107602069164,(#H7:::0.48271071306160374,(4,((1)#H7:::0.5172892869383963,2):5.872857349819618):0.5701459078518265):0.0);, with -loglik 170.76491232312355
(1,#H7:0.2619732365377584::0.2147015688910482,((4,((3,(6,5):0.3625007066527544):0.0)#H7:0.9818429896871675::0.7852984311089518):1.3066562802202648,2):4.366729495433193);, with -loglik 176.67939122045223
