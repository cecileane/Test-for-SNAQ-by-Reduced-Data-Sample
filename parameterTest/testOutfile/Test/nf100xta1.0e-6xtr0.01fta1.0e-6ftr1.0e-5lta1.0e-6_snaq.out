(3,(6,5):0.335347317849328,((1,(2)#H7:::0.6269753195260324):2.155860132049398,(4,#H7:::0.3730246804739677):2.155860132049398):0.12573310842051721); -Ploglik = 155.49625734551452
 Dendroscope: (3,(6,5):0.335347317849328,((1,(2)#H7):2.155860132049398,(4,#H7):2.155860132049398):0.12573310842051721);
 Elapsed time: 15.470552934 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,(6,5):0.335347317849328,((1,(2)#H7:::0.6269753195260324):2.155860132049398,(4,#H7:::0.3730246804739677):2.155860132049398):0.12573310842051721);, with -loglik 155.49625734551452
 ((3,(1,(2,#H7:::0.39462115396102704):10.0):1.130042730052587):0.22279535041049814,6,(5,(4)#H7:::0.605378846038973):0.06848863698245954);, with -loglik 169.89572985762143
-------
